server/routes/api.js

```diff
@@ -34,7 +35,12 @@ router.get('/usuario', async (req, res) => {
  */
 router.get('/usuario/:username', async (req, res) => {
   const username = req.params.username;
-  res.status(200).json(await UsuarioService.consultaUsuario(username));
+  try {
```

Comment:
Coloca tudo em um mesmo try/catch. E retorna o erro dependendo do catch.

See this: https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Statements/try...catch